src/components/xlsx.js

```diff
@@ -1,133 +0,0 @@
-import Excel from 'exceljs';
-import stream from 'stream';
-
-const writerWorkbookOptions = [
-  'creator',
-  'lastModifiedBy',
-  'created',
-  'modified'
-];
-
-class XLSXParser extends stream.Transform {
-  constructor(options, transform) {
-    super({objectMode: true});
-
-    this.workbook = new Excel.Workbook();
-    this.stream = this.workbook.xlsx.createInputStream();
-    this.ondone = null;
-    this.options = options;
-    this.transform = transform;
```

Comment:
`this.transform`. I changed it into `this.transformObject` to avoid the confusion.